query-service/src/integrationTest/resources/config-bootstrapper/attribute-service/hypertrace/hypertrace_api_attributes.conf

```diff
@@ -0,0 +1,192 @@
+version = 2
```

Comment:
This is a copy of the config and would be hard to maintain. And actual data can change.

Reply:
Makes sense. Let me see what can be done here.